packages/flame/lib/src/events/messages/drag_start_event.dart

```diff
@@ -34,4 +34,9 @@ class DragStartEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'DragStartEvent(pointerId: $pointerId, deviceKind: $deviceKind, '
```

Comment:
```suggestion
      '$DragStartEvent(pointerId: $pointerId, deviceKind: $deviceKind, '
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.